packages/front-end/pages/saved-groups.tsx

```diff
@@ -87,13 +87,7 @@ export default function SavedGroupsPage() {
                         className="d-none d-md-table-cell text-truncate"
                         style={{ maxWidth: "100px" }}
                       >
-                        {s.values.map((attribute, index) => {
-                          if (index === s.values.length - 1) {
-                            return attribute;
-                          } else {
-                            return `${attribute}, `;
-                          }
-                        })}
+                        {String(s.values)}
```

Comment:
This is completely optional, and also predicated on us liking the solution of using the `String()` constructor as a function. This just adds a layer of abstraction that makes the code arguably easier to read.